doc/nrf/gs_modifying.rst

```diff
@@ -213,14 +223,19 @@ Changes are picked up immediately, and you do not need to re-open the project in
 While it is possible to edit the :file:`.config` file directly, you should use SES or a tool like menuconfig or guiconfig to update it.
 These tools present all available options and allow you to select the ones that you need.
 
+To edit the file in |VSC|, use the nRF Kconfig extension.
+This extension organizes the Kconfig option in a hierarchical list and lets you select and deselect the desired options.
+To save the changes made using the nRF Kconfig extension, click the :guilabel:`Save` button.
```

Comment:
We have three different save options actually. Are we assuming here that the user is selecting their kconfig options for the first time and needs to just save them to a new .conf file?

Reply:
I'm using the differentiation mentioned in the VSC docs. As we're talking about temporary changes, I'm mentioning "Save". In line 264, where we mention permanent changes, I mention "Save to file".